### PR TITLE
Adds react native sign out guide

### PR DESCRIPTION
**Pull request by NikitaAvraimov-okta**
## Description:
- **What's changed?** Added react native section to sign out guides https://developer.okta.com/docs/guides/sign-users-out/android/before-you-begin/
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-233801](https://oktainc.atlassian.net/browse/OKTA-233801)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/define-signout-callback/react-native/addbaseuri.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+Signing out of Okta requires the app to open a browser and navigate to the [end session endpoint](https://developer.okta.com/docs/reference/api/oidc/#logout). Okta destroys the user's session and immediately redirects back to your application. To do this, you must define a callback route for the sign-out process.
+
+Make sure that the `endSessionRedirectUri` configured at `createConfig` is the same as the one defined in the **Logout redirect URI** section. For example if you are initializing your client via a config file:
```
Comment (JayNewstrom) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/define-signout-callback/react-native/addbaseuri.md:3]: Might be worth adding a little more context here: "**Logout redirect URI** section of the okta admin dashboard" or similar.

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/define-signout-callback/react-native/addbaseuri.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+Signing out of Okta requires the app to open a browser and navigate to the [end session endpoint](https://developer.okta.com/docs/reference/api/oidc/#logout). Okta destroys the user's session and immediately redirects back to your application. To do this, you must define a callback route for the sign-out process.
```
Comment (JayNewstrom) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/define-signout-callback/react-native/addbaseuri.md:1]: I see you added a note about this being browser based flow only in another file. Do we want the same here?

**Review by JayNewstrom — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/define-signout-callback/react-native/addbaseuri.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+If you'd like to use a custom sign-out flow, you can skip to the next step: <GuideLink link="../sign-out-of-okta">Sign users out of Okta</GuideLink>.
+
+If you'd like to use the browser sign-out flow, you will need to define a callback route.
+
+Make sure that the `endSessionRedirectUri` configured at `createConfig` is the same as the one defined in the **Logout redirect URI** section of the okta admin dashboard. For example if you are initializing your client via a config file:
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/define-signout-callback/react-native/addbaseuri.md:5]:
```suggestion
Make sure that the `endSessionRedirectUri` configured at `createConfig` is the same as the one defined in the **Logout redirect URI** section of the Okta Admin Console. For example if you are initializing your client with a config file:
```

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/next-steps/react-native/samples.md [hunk 4]
```diff
@@ -0,0 +1 @@
+The samples available in our [React Native samples repo](https://github.com/okta/samples-js-react-native) on GitHub show how to sign users out of the app and out of Okta.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/next-steps/react-native/samples.md:1]:
```suggestion
The samples available in our [React Native samples repo](https://github.com/okta/samples-js-react-native) on GitHub show how to sign users out of both the app and Okta.
```

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/react-native/remotesignout.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
+### Clear the Okta session
+
+Clear the browser session and clear the app session (stored tokens) in memory. Fires an event once a user successfully logs out.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/react-native/remotesignout.md:3]:
```suggestion

Next you should clear the browser session and app session (stored tokens) in memory. An event will fire once a user successfully logs out.
```

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/react-native/remotesignout.md [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+
+> Note: This method apply for browser-sign-in scenario only. Use a combination of revokeToken (optional) and clearTokens methods to signOut when use custom-sign-in.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/react-native/remotesignout.md:5]:
```suggestion

> Note: This method applies only to the browser sign-in scenario. Use a combination of `revokeToken` (optional) and `clearTokens` methods to sign out when using custom sign-in.
```

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/react-native/remotesignout.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+
+browser-sign-in example:
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/react-native/remotesignout.md:7]:
```suggestion
Browser-sign-in example:
```

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-your-app/react-native/localsignout.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+import { clearTokens } from '@okta/okta-react-native';
+
+clearTokens()
+    .then(() => { })
+    .catch(e => { });
+```
+
+#### Revoke tokens (optional)
+
+Calling `clearTokens()` discards tokens from local device storage, but they are technically still active until they expire. An optional step is to revoke the tokens so they can't be used, even by accident. You can revoke tokens using following requests:
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-your-app/react-native/localsignout.md:13]:
```suggestion
Calling `clearTokens()` discards tokens from the local device storage, but they are technically still active until they expire. An optional step is to revoke the tokens so they can't be used, even by accident. You can revoke tokens using the following request:
```

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-your-app/react-native/localsignout.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+
+```javascript
+await revokeAccessToken();
+```
+
+Revoke the access token to make it inactive. Resolves true if access token has been successfully revoked.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-your-app/react-native/localsignout.md:19]:
```suggestion
This revokes the access token to make it inactive. Resolves `true` if access token has been successfully revoked.
```

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-your-app/react-native/localsignout.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+
+```javascript
+await revokeIdToken();
+```
+
+Revoke the identity token to make it inactive. Resolves true if id token has been successfully revoked.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-your-app/react-native/localsignout.md:25]:
```suggestion
This revoke the ID token to make it inactive. Resolves `true` if ID token has been successfully revoked.
```

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-your-app/react-native/localsignout.md [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+
+```javascript
+await revokeRefreshToken();
+```
+
+Revoke the refresh token to make it inactive. Resolves true if refresh token has been successfully revoked.
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-your-app/react-native/localsignout.md:31]:
```suggestion
This revokes the refresh token to make it inactive. Resolves `true` if refresh token has been successfully revoked.
```

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/react-native/remotesignout.md [hunk 12]
```diff
@@ -0,0 +1,27 @@
...
+
+Browser-sign-in example:
+
+```javascript
+import { signOut } from '@okta/okta-react-native';
+
+signOut();
+```
+
+custom-sign-in example:
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/react-native/remotesignout.md:17]:
```suggestion
Custom-sign-in example:
```

File: packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/react-native/remotesignout.md [hunk 13]
```diff
@@ -0,0 +1,27 @@
+### Clear the Okta session
+
+
+Next you should clear the browser session and app session (stored tokens) in memory. An event will fire once a user successfully logs out.
+
+
+> Note: This method applies only to the browser sign-in scenario. Use a combination of `revokeToken` (optional) and `clearTokens` methods to sign out when using custom sign-in.
+
+Browser-sign-in example:
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/react-native/remotesignout.md:9]:
```suggestion
Browser sign-in example:
```

**Review by jakubvul-okta — APPROVED** [review]

**Pull request merged**